composer.json

```diff
@@ -37,7 +37,7 @@
         "php-http/guzzle6-adapter": "^1.1.1",
         "php-http/react-adapter": "^0.2.1",
         "php-http/buzz-adapter": "^0.3",
-        "symfony/symfony": "^2.8 || ^3.0",
```

Comment:
Sounds fair to me to suggest both packages and adding them to require-dev. Looks better than depending on symfony/symfony.